app/controllers/submissions_controller.rb

```diff
@@ -120,6 +136,28 @@ def destroy
                                                    @submission.assessment)) && return
   end
 
+  action_auth_level :destroyBatch, :instructor
+  def destroyBatch
+    submission_ids = params[:submission_ids]
+    submissions = submission_ids.map { |sid| @assessment.submissions.find_by(id: sid) }
+    scount = 0
+    fcount = 0
+    submissions.each do |s|
+      if s.destroy
+        scount += 1
+      else
+        fcount += 1
+      end
+    end
+    if fcount == 0
+      flash[:success] = "#{scount} submission(s) destroyed. #{fcount} submission(s) failed."
```

Comment:
nit: use the built-in ruby pluralize function instead of having the `(s)`:

`#{"submission".pluralize(scount)}`

Reply:
fixed.